src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CompilationContext.cs

```diff
@@ -296,17 +288,25 @@ private static string GetNextMethodName(ArrayBuilder<MethodSymbol> builder)
                     if (!argumentsOnly)
                     {
                         // Pseudo-variables: $exception, $ReturnValue, etc.
-                        if (aliases.Count > 0)
+                        var aliasNames = PooledHashSet<string>.GetInstance(); // To de-dup $ReturnValue/$returnvalue, etc.
```

Comment:
How do we get duplicate alias names? And if we do, is it necessary to remove duplicates?